boards/tests/test_views.py

```diff
@@ -38,3 +39,21 @@ def test_topic_list_view_produces_status_code_200(self):
         request = self.factory.get(reverse("all_topics"))
         response = board_list(request)
         self.assertEquals(200, response.status_code)
+
+
+class BoardDetailView(TestCase):
+    """ It tests the views of BoardDetailView"""
+
+    def setUp(self):
+        self.board_obj = Board.objects.create(
+            name="1st", slug="slug-of-board1", description="board_obj"
+        )
+        self.factory = RequestFactory()
+
+    def test_board_detail_is_refrencing_to_correct_associated_function(self):
+        view = resolve("/slug_of_board1/")
+        self.assertEquals(view.func, board_detail)
+
+    def test_board_detail_view_produces_status_code_200(self):
+        response = self.client.get(reverse("detail", args=(self.board_obj.slug,)))
```

Comment:
I tried a lot using RequestFactory but it was not working so I had to do it with Testcase client. I am trying again to make RequestFactory work